OmniKit/MessageTransport/MessageBlocks/PodInfoActivationTime.swift

```diff
@@ -8,23 +8,26 @@
 
 import Foundation
 
-public struct PodInfoFault : PodInfo {
+// Type 5 PodInfo returns the pod activation time, time pod alive, and the possible fault code
+public struct PodInfoActivationTime : PodInfo {
     // OFF 1  2  3  4 5  6 7 8 9 10111213 1415161718
     // DATA   0  1  2 3  4 5 6 7 8 9 1011 1213141516
     // 02 11 05 PP QQQQ 00000000 00000000 MMDDYYHHMM
 
-    public var podInfoType: PodInfoResponseSubType = .fault
+    public let podInfoType: PodInfoResponseSubType = .activationTime
     public let faultEventCode: FaultEventCode
     public let timeActivation: TimeInterval
     public let dateTime: DateComponents
     public let data: Data
     
     public init(encodedData: Data) throws {
         
-        if encodedData.count < 16 {
+        guard let podInfoType = PodInfoResponseSubType.init(rawValue: encodedData[0]), podInfoType == self.podInfoType else {
+            throw MessageError.unknownValue(value: encodedData[0], typeDescription: "activationTime")
```

Comment:
typeDescription: "PodInfoResponseSubType"

Reply:
Changed all PodInfoResponseSubType.init() uses to PodInfoResponseSubType()